lib/imgbundler/imgbundler.go

```diff
@@ -160,10 +160,10 @@ func worker(ctx context.Context, ms *xmain.State, href []byte, isRemote bool) ([
 	var mimeType string
 	var err error
 	if isRemote {
-		ms.Log.Debug.Printf("fetching %s remotely", string(href))
+		log.Debug(ctx, fmt.Sprintf("fetching %s remotely", string(href)))
```

Comment:
I don't think this is the equivalent log stream? I think you should make a tiny interface with just Debug and Info and pass in a logger implementation